.github/workflows/terraform-github.yml

```diff
@@ -24,28 +24,49 @@ jobs:
   github-plan-and-apply:
     runs-on: ubuntu-latest
     env:
-      AWS_ACCESS_KEY_ID:  ${{ secrets.AWS_ACCESS_KEY_ID }}
-      AWS_SECRET_ACCESS_KEY:  ${{ secrets.AWS_SECRET_ACCESS_KEY }}
       TF_VAR_github_token: ${{ secrets.TERRAFORM_GITHUB_TOKEN }}
       TF_IN_AUTOMATION: true
+      AWS_REGION: "eu-west-2"
+      ENVIRONMENT_MANAGEMENT: ${{ secrets.MODERNISATION_PLATFORM_ENVIRONMENTS }}
```

Comment:
you can also define these outside of the job so that it is global (although it is not necessary at this stage, since there is only one job)